content/blog/2023/09/camel-quarkus-release-3.2.0/index.md

```diff
@@ -0,0 +1,27 @@
+---
+title: "Camel Quarkus 3.2.0 Released"
+date: 2023-09-15
+authors: ["zbendhiba"]
+categories: ["Releases", "Camel Quarkus"]
+preview: "Camel Quarkus 3.2.0 LTS Released"
+summary: "Camel Quarkus 3.2.0 is the first final 3.x major release"
+---
+
+<sub><sup>Image by <a href="https://www.flickr.com/photos/lanzen/5984113332">Anders Lanzen</a> <a href="https://creativecommons.org/licenses/by-nc-sa/2.0">CC BY-NC-SA 2.0</a></sup></sub>
+
+We are pleased to announce the 3.2.0 release of Camel Quarkus. This is the first major 3.x release.
+
+Camel Quarkus 3.2.0 is an LTS release, it aligns Quarkus 3.2.x LTS and Camel 4.0.x LTS releases.
+
+## Camel 4.0.0
+Please refer to the [Camel 4.0.0 announcement](/blog/2023/08/RELEASE-4.0.0/) for more details.
+
+## Quarkus 3.2.5.Final
+Camel Quarkus 3.2.0 is based on Quarkus [3.2.5.Final](https://quarkus.io/blog/quarkus-3-2-5-final-released/). However, it's important to note that this version is available in the Quarkus platform starting from Quarkus 3.2.6.Final. Therefore, users are encouraged to upgrade to Quarkus 3.2.6.Final for the latest features and improvements.
+
```

Comment:
Maybe a proposal as below:

Camel Quarkus 3.2.0 is based on Quarkus [3.2.5.Final](https://quarkus.io/blog/quarkus-3-2-5-final-released/). However, it's important to note that this version is available in Quarkus platform 3.2.6.Final onward. Therefore, users are encouraged to upgrade to Quarkus Platform 3.2.6.Final for the latest features and improvements.

Reply:
yeah thansk @aldettinger.  I like that proposal. Let me update it this quickly.